Services/AzineMailgunTwigExtension.php

```diff
@@ -1,10 +1,22 @@
 <?php
 namespace Azine\MailgunWebhooksBundle\Services;
 
+use Azine\MailgunWebhooksBundle\DependencyInjection\AzineMailgunWebhooksExtension;
+use Azine\MailgunWebhooksBundle\Entity\Repositories\MailgunEventRepository;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class AzineMailgunTwigExtension
+ * Provides some filters and global variables
+ * @package Azine\MailgunWebhooksBundle\Services
+ */
 class AzineMailgunTwigExtension extends \Twig_Extension
 {
-    public function __construct()
+    private $container;
+    
+    public function __construct(ContainerInterface $container)
```

Comment:
Injecting the ContainerInterface IMHO is bad practice. Please inject only/exactly what you need. This can be solved elegantly in the services.yml
Also the the answer to: https://stackoverflow.com/questions/23931321/in-symfony2-why-is-it-a-bad-idea-to-inject-the-service-container-rather-than-i